about/system_requirements.rst

```diff
@@ -239,21 +233,14 @@ Desktop or laptop PC - Minimum
 +----------------------+-----------------------------------------------------------------------------------------+
 | **Storage**          | 150 MB (used for the executable, project files and cache)                               |
 +----------------------+-----------------------------------------------------------------------------------------+
-| **Operating system** | - **For native exports:** Windows 7 (Compatibility and Forward+/Mobile, Vulkan),        |
-|                      |   Windows 10 (Forward+/Mobile, Direct3D 12), macOS 10.13 (Compatibility), macOS 10.15   |
+| **Operating system** | - **For native exports:** Windows 10, macOS 10.13 (Compatibility), macOS 10.15          |
 |                      |   (Forward+/Mobile, Vulkan), macOS 13.0 (Forward+/Mobile, Metal), Linux distribution    |
-|                      |   released after 2016                                                                   |
+|                      |   released after 2018                                                                   |
 |                      | - **For web exports:** Firefox 79, Chrome 68, Edge 79, Safari 15.2, Opera 64            |
```

Comment:
I'm not sure this one is aging well.

CC @adamscott @Calinou 

Maybe we should just say:

> Recent mainstream versions of the follower browsers: Firefox (including ESR), Chrome, Edge, Safari, Opera.

But there's other browsers which likely work fine like Vivaldi, or Firefox rebanded versions like Waterfox.

Maybe we should go with something like:

> Recent versions of mainstream browsers: Firefox and derivatives (including ESR), Chrome and Chromium derivatives, Safari and WebKit derivatives.

Reply:
@bruvzg Can you apply the above change? This should be good to merge afterwards.

Reply:
Done.